backend/config/routes.rb

```diff
@@ -139,6 +139,7 @@
       collection do
         get :search
       end
+      resource :icon, only: :destroy, to: "taxons#clear_icon"
```

Comment:
We don't have a proper `Icon` resource, the resource involved here is a `Taxon`, and the controller method was added to TaxonsController, so I think it may be more appropriate to add the `clear_action` route to the `taxon` resource... what do you think?